### PR TITLE
Bump containernetworking dependencies only

### PR DESCRIPTION
**Pull request by mogren**
*Description of changes:*
* Bump of containernetworking dependencies
* The `ns` package has moved to the `plugins` package.
* Bumped portmap version

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/ipamd/ipamd.go [hunk 1]
```diff
@@ -150,13 +150,13 @@ var (
...
 			Name: "awscni_add_ip_req_count",
-			Help: "The number of add IP address request",
+			Help: "The number of add IP address requests",
 		},
 	)
 	delIPCnt = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Name: "awscni_del_ip_req_count",
-			Help: "The number of delete IP address request",
+			Help: "The number of delete IP address requests",
```
Comment (astrived) [inline pkg/ipamd/ipamd.go:159]: Is this change primarily for readability ?

File: pkg/cninswrapper/ns.go [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 package cninswrapper
 
-import "github.com/containernetworking/cni/pkg/ns"
+import "github.com/containernetworking/plugins/pkg/ns"
```
Comment (astrived) [inline pkg/cninswrapper/ns.go:16]: Have all these packages moved under plugins directory now ? Looking at this PR in isolation is not giving me that insight. Can we still continue to use cni directory packages ?

File: pkg/ipamd/ipamd.go (hunk 1 above)
Comment (mogren) [reply pkg/ipamd/ipamd.go:159]: Yes, sentence was not correct.

File: pkg/cninswrapper/ns.go (hunk 2 above)
Comment (mogren) [reply pkg/cninswrapper/ns.go:16]: The upstream package was moved in [v0.6.0 back in August 2017](https://github.com/containernetworking/cni/releases/tag/v0.6.0), so we have to import the files from where they are now.

Comment (astrived) [reply pkg/cninswrapper/ns.go:16]: Thanks for clarification. Lets run integration changes post this at least once again before the release.

File: go.mod [hunk 3]
```diff
@@ -15,7 +16,7 @@ require (
 	github.com/google/gofuzz v0.0.0-20170612174753-24818f796faf // indirect
 	github.com/googleapis/gnostic v0.2.0 // indirect
 	github.com/gregjones/httpcache v0.0.0-20190212212710-3befbb6ad0cc // indirect
-	github.com/hashicorp/golang-lru v0.0.0-20180201235237-0fb14efe8c47 // indirect
```
Comment (jaypipes) [inline go.mod:18]: \o/ always good to get rid of older non-semver deps.

**Pull request merged**